src/Sentry/SentryOptions.cs

```diff
@@ -1008,6 +1008,11 @@ public bool JsonPreserveReferences
     /// </summary>
     internal bool InitNativeSdks { get; set; } = true;
 
+    /// <summary>
+    /// Idling transaction timeout after which transaction will be finished automatically.
+    /// </summary>
+    internal TimeSpan? IdleTimeout { get; set; }
+
```

Comment:
```suggestion
```
This seems to cause a lot of confusion. Let's remove it and whenever auto instrumentation gets implemented we'll add it back in.